server/cli/flags.go

```diff
@@ -105,6 +105,10 @@ var flagUsage = map[string]string{
         ranges. The scan is slowed as necessary to approximately achieve this
         duration.
 `,
+	"scan-max-idle-time": `
```

Comment:
Oh. If we don't have --scan-interval we probably don't want to have --scan-max-idle-time either, right?